geometry/clipping.cpp

```
@@ -206,6 +206,7 @@ vector<m2::SharedSpline> ClipSplineByRect(m2::RectD const & rect, m2::SharedSpli
   case RectCase::Inside: return {spline};
   case RectCase::Outside: return {};
   case RectCase::Intersect: return ClipPathByRectImpl(rect, spline->GetPath());
+  default: return {};
```

No, it will silence the error if a new switch case will be added.